proto/substrait/algebra.proto

```diff
@@ -228,6 +228,71 @@ message AggregateRel {
   }
 }
 
+// The Window relational operate on a group of rows, referred to as a window, and calculate a return value for each row based on the group of rows.
+message WindowRel {
+  RelCommon common = 1;
+  Rel input = 2;
+  repeated WindowRelFunction window_functions = 3;
+  repeated Expression partition_expressions = 4;
+  repeated SortField sorts = 5;
+
+  substrait.extensions.AdvancedExtension advanced_extension = 10;
+
+  // A window rel function call.
+  message WindowRelFunction {
+    // Points to a function_anchor defined in this plan, which must refer
+    // to a window function in the associated YAML file. Required; 0 is
+    // considered to be a valid anchor/reference.
+    uint32 function_reference = 1;
+
+    // The arguments to be bound to the function. This must have exactly the
```

Comment:
Why have we removed the all/distinct property from this function?

Reply:
I don't understand what you're saying about `all/distinct property` here.

Reply:
This is below at:

> AggregateFunction.AggregationInvocation invocation = 10;

I don't think it is missing.